components/jupyter-web-app/backend/kubeflow_jupyter/common/api.py

```diff
@@ -101,6 +101,17 @@ def get_notebooks(ns):
     )
 
 
+def get_poddefaults(ns):
+    return wrap_resp(
+        'poddefaults',
```

Comment:
just curious: the pattern here seems to be using single quote for first param, and double quote for third and later ones. What's the reason?